src/host.rs

```diff
@@ -120,6 +120,30 @@ struct FlashOpts {
     flexspi: FlexSpi,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+struct EnvOverride {
+    default: usize,
+    env: Option<String>,
+}
+
+impl EnvOverride {
+    const fn new(default: usize) -> Self {
+        Self { default, env: None }
+    }
+    fn set_env_key(&mut self, key: String) {
+        println!("cargo:rerun-if-env-changed={key}");
+        self.env = Some(key);
+    }
+    fn read(&self) -> Result<usize, Box<dyn std::error::Error>> {
+        if let Some(val) = self.env.as_ref().and_then(|key| env::var(key).ok()) {
+            let val = val.parse()?;
+            Ok(val)
```

Comment:
You seem to have dropped the ability to specify "8K" as the stack size.  I'm fine with that, I just wanted to be sure it was intentional.  I don't think typing "8192" is much harder.

Reply:
I see that it's convenient. It's implemented, tested, and documented as being multiples of 1024 bytes.